### DISCUSSION TITLE
Blue Iris & D800 problems

### DISCUSSION THREAD

**Discussion by xnshady** [thirtythreeforty/neolink#264] [General]
Hi.
Im currently trying to get the stream from my D800 into Blue Iris. And it will not connect....
My neolink connects fine to the camera

Here is neolink
![277404692_1033155633944703_1264634888546279618_n](https://user-images.githubusercontent.com/10730810/161053065-e6aed347-192a-449b-b69a-b5ccb96efb95.png)

Here is my Blue Iris Settings:
![277722014_674024820572101_3085343388303835881_n](https://user-images.githubusercontent.com/10730810/161053709-533cbb62-79de-4b74-90ed-1163acc5e1b2.png)

Here is my Config file:
![277521200_780510602926330_3231134475828818973_n](https://user-images.githubusercontent.com/10730810/161054975-a874df96-d87e-4234-aba7-a7b21fdb257a.png)

Any suggestions where to look ? 
-RM

**Comment by QuantumEntangledAndy** [comment]
I cannot see anything up with the config, can you get it to work outside of BI. Maybe using `ffplay`

p.s. I also assume that you know what `127.0.0.1` and that this is indeed correct.
